pkg/controller/subnet.go

```diff
@@ -1635,6 +1635,17 @@ func calcDualSubnetStatusIP(subnet *kubeovnv1.Subnet, c *Controller) error {
 		return err
 	}
 
+	usingIPNums := len(podUsedIPs.Items)
+	for _, podUsedIP := range podUsedIPs.Items {
```

Comment:
```suggestion
	for _, podUsedIP := range subnet.Spec.ExcludeIps {
```
我在 master 将 Spec.ExcludeIps 放在外层循环了，感觉会更好一些。

Reply:
这个逻辑，是遍历所有的 IPs，查看是否在 excludeIPs 里边。
如果是 excludeIPs 循环在外，IPs 遍历在内，那查到之后，break 的逻辑，就不对了。break 就不会再查接下来的 IPs，而是重新检查下一个 excludeIPs。这个并不是期望的结果

Reply:
fixed

Reply:
这里的逻辑，是遍历每一个 IP cr，确认是否在 subnet.Spec.ExcludeIPs 中，如果在，则减去计数
如果调换循环顺序，就成为了检查 excludeIPs 是否包含 IP，这个时候 break 满足条件退出的循环，就忽略了剩余 IP cr 的检查。
例如 subnet.spec.excludeIPs 设置范围  - 12.16.10.10..12.16.10.12，然后创建两个固定IP的 pod，12.16.10.10和 12.16.10.12。按照这个逻辑检查的时候，检查12.16.10.10 符合要求，就退出循环，会导致 12.16.10.12 这个计数错误

Reply:
恩恩 我去改下